Feat/cfg edgetpucfg test by hyeonuk

Hyeon-Uk opened the pull request.

Hyeon-Uk committed:
[Test] Add sorted test

- add sorts config test

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

Hyeon-Uk committed:
[Test] Add isSame() test

- add isSame() test

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

Hyeon-Uk committed:
[EdgeTPU] add updateSectionWithValue() tests

- positive test complete
- negative test complete

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

Hyeon-Uk committed:
[Test] implements updateSectionWithKeyValue() test

- positive test complete
- negative test complete

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

hohee-hee commented on src/Tests/CfgEditor/EdgeTpuCfgData.test.ts:
```diff
@@ -102,5 +102,160 @@ suite("EdgetpuCfgEditor", function () {
+        assert.strictEqual(cfg["edgetpu-compile"]["show_operations"], "True");
+      });
+
+      test("NEG: try to update 'intermediate_tensors' and 'search_delegate' togather", function () {
```
A typo here, "togather"

Hyeon-Uk replied:
oh, my mistake. thanks!

Hyeon-Uk committed:
[Fix] fix typo

- togather -> together

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

Hyeon-Uk committed:
Merge branch 'feat/cfg-edgetpucfg-test' into feat/cfg-edgetpucfg-test-hyeonuk
